core/core.js

```diff
@@ -0,0 +1,164 @@
+'use strict'; // For testing with node.js
+
+// Global scope declarations:
+var $ = Object.create(Object.prototype);
```

Comment:
I was attempting to standardise on Object.create for all object creation, but perhaps in this case that is not really necessary.